src/components/CovMap.tsx

```diff
@@ -168,14 +165,30 @@ export const CovMap = () => {
   const mappingLayers = Object.keys(visual.mappings);
   const handleMapClick = (pointerEvent, stateViewport) => {
     const { features } = pointerEvent;
+    console.log(features)
     if (features.length > 0) {
+
+      /* handle multiple features. this happens when a street or text is clicked */
+      let countyFeatures
+      if (features.length > 1) {
+        // find out target features from index.ts
+        const layers = config.visuals.covmap.layers;
+        if (!layers) return;
+        const fillLayer = layers.filter(layer => layer.type === LayerType.FILL); // get the layer with the layer type "fill"
+        if (!fillLayer) return;
+        countyFeatures = features.filter(feature => fillLayer.some(layer => feature.layer && (feature.layer.id === layer.id)));
```

Comment:
Vorher war das etwas praktischer, so dass man in der config ein Array haette angeben koennen mit den Layer IDs welche klickbar sein sollen?